wire-desktop/PKGBUILD

```diff
@@ -30,27 +29,25 @@ build() {
 package() {
   # Place files
   install -d "${pkgdir}/usr/lib/${pkgname}"
-  cp -a "${srcdir}/${pkgname}-linux-${pkgver}"/wrap/dist/linux*unpacked/* "${pkgdir}/usr/lib/${pkgname}"
-  
-  # Symlink main binary
-  install -d "${pkgdir}/usr/bin"
-  ln -s "/usr/lib/${pkgname}/${pkgname}" "${pkgdir}/usr/bin/${pkgname}"
-  
+  cp -a "${pkgname}-linux-${pkgver}"/wrap/dist/linux-unpacked/resources/app/* "${pkgdir}/usr/lib/${pkgname}"
+
+  # Place launcher script
+  install -Dm755 -t "${pkgdir}/usr/bin/" "${pkgname}"
+
   # Place desktop entry and icon
-  desktop-file-install -m 644 --dir "${pkgdir}/usr/share/applications/" "${srcdir}/${pkgname}.desktop"
+  desktop-file-install -m 644 --dir "${pkgdir}/usr/share/applications/" "${pkgname}.desktop"
+  local res
   for res in 32x32 256x256; do
-    install -dm755 "${pkgdir}/usr/share/icons/hicolor/${res}/apps"
-    install -Dm644 "${srcdir}/${pkgname}-linux-${pkgver}/resources/icons/${res}.png" \
-      "${pkgdir}/usr/share/icons/hicolor/${res}/apps/${pkgname}.png"
+    install -Dm644 "${pkgname}-linux-${pkgver}/resources/icons/${res}.png" "${pkgdir}/usr/share/icons/hicolor/${res}/apps/${pkgname}.png"
   done
 
   # Spellcheck dictionaries
-  rm -rf "${pkgdir}/usr/lib/${pkgname}/resources/app/node_modules/spellchecker/vendor/hunspell_dictionaries"
-  ln -s "/usr/share/hunspell" "${pkgdir}/usr/lib/${pkgname}/resources/app/node_modules/spellchecker/vendor/hunspell_dictionaries"
+  rm -rf "${pkgdir}/usr/lib/${pkgname}/node_modules/spellchecker/vendor/hunspell_dictionaries"
+  ln -s "/usr/share/hunspell" "${pkgdir}/usr/lib/${pkgname}/node_modules/spellchecker/vendor/hunspell_dictionaries"
 
   # Place license files
+  local license
   for license in "LICENSE.electron.txt" "LICENSES.chromium.html"; do
```

Comment:
Without specifying `local`, the loop variable was overriding `license` in the top of the file 😱